
We need to have a shutdown group to allow the shutdown icon
to work correctly. Any users that want to use shutdown like
the xuser should be added to this group.

Upstream-Status: Inappropriate [Embedded]

Signed-off-by: Saul Wold <sgw@linux.intel.com>
Index: base-passwd-3.5.26/group.master
===================================================================
--- base-passwd-3.5.26.orig/group.master
+++ base-passwd-3.5.26/group.master
@@ -36,5 +36,6 @@ sasl:*:45:
 plugdev:*:46:
 staff:*:50:
 games:*:60:
+shutdown:*:70:
 users:*:100:
 nogroup:*:65534:
